HandleNHNotificationCallActivity/handler.ts

```diff
@@ -98,9 +101,22 @@ export const getCallNHServiceActivityHandler = (
             retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
           );
         case NotifyKind.Notify:
-          return notify(message.installationId, message.payload).mapLeft(e =>
-            retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
-          );
+          return notify(message.installationId, message.payload)
+            .mapLeft(e =>
+              retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
+            )
+            .chainFirst(
+              taskEither.of(
+                telemetryClient.trackEvent({
+                  name: "api.messages.create.push",
+                  properties: {
+                    isSuccess: "true",
+                    messageId: message.payload.message_id
+                  },
+                  tagOverrides: { samplingEnabled: "false" }
+                })
+              )
+            );
         case DeleteKind.DeleteInstallation:
```

Comment:
Why track only notify events?

Reply:
that's the latest step of https://github.com/pagopa/io-functions-services/pull/100